app/src/main/res/values/strings.xml

```diff
@@ -4,6 +4,8 @@
         1. You are supposed to make as many words as you can from the given set of letters in the grid.
         \n2. If the submitted word is a valid one then it is added to the list and the score would be updated.
         \n3. The minimum valid word length is 4 i.e. "cars" is a valid word whereas "car" is not.
+        \n4. You can select letters joining currently selected letters only.
```

Comment:
...you can select letters such that consecutive letters are neighbour to one another